File: contents/cgi_files/GET.cpp

#include "iostream"
#include "string"

int main()
{
	std::string str;

	str = "Content-length: 100\r\n";
	str += "Content-type: text/plane\r\n";
	str += "\r\n";
	str += "Get Wild";
	std::cout << str;
}

File: contents/cgi_files/GET.py
#!/usr/bin/python3
import sys

sys.stdout.write('Content-type: text/css\r\n')
sys.stdout.write('COntent-length: 25\r\n')
sys.stdout.write('\r\n')
sys.stdout.write('GET python\n')
sys.stdout.write('May the Force be with you.')